s3backgrounddelete/s3backgrounddelete/cortx_s3_config.py

```diff
@@ -353,6 +353,14 @@ def get_max_keys(self):
         else:
             return 1000
 
+    def get_threshold(self):
+        """Return the threshold for max."""
+        threshold = self.s3confstore.get_config('indexid>threshold')
```

Comment:
Why is threshold value inside indexid and not inside message_bus? @vimalraghwani20 what are your thoughts?

Reply:
> Why is threshold value inside indexid and not inside message_bus? @vimalraghwani20 what are your thoughts?

Initially i was also thinking the same to move to msgbus but then i thought of keeping it near max_key so that who ever changes the max_key can atleast look at threshold value 
If we all agree we can move this to msgbus section too.

Reply:
confirmed with nilesh, this seems fine according to him.